lib/__test__/utils.test.js

```diff
@@ -350,3 +338,43 @@ describe('test get file diffs', () => {
         expect(result['otherFile.js']).toEqual(mockOtherFileDiff);
     });
 });
+
+describe('test caching glob calls', () => {
```

Comment:
thanks for adding tests! It looks like they're not yet run by a github action?

Reply:
I added the github actions in the other PR #2

Reply:
oh interesting. Can you change the base of this PR to be `WEB-2645`? Then we should get the checks here

Reply:
Okay, I've updated the base of the PR and got the checks to run and mostly pass, but the two tests that are failing right now are failing on really strange exit codes and I'm not quite sure why. The autofix test is throwing an error.

When I run `node ./node_modules/actions-utils/list-changed-files.js | grep '.*\.js$' | xargs npm run -s format-files` in my terminal, it says `error: malformed object name HEAD~20`. That line comes from the first command run in the pipeline.